plugins/newsdownloader.koplugin/main.lua

```diff
@@ -135,32 +136,75 @@ function NewsDownloader:loadConfigAndProcessFeeds()
             UIManager:show(info)
             -- processFeedSource is a blocking call, so manually force a UI refresh beforehand
             UIManager:forceRePaint()
-            self:processFeedSource(url, tonumber(limit))
+            self:processFeedSource(url, tonumber(limit), unsupported_feeds_urls)
             UIManager:close(info)
         else
             logger.warn('NewsDownloader: invalid feed config entry', feed)
         end
     end
 
-    UIManager:show(InfoMessage:new{
-        text = _("Downloading news finished."),
-        timeout = 1,
-    })
+    if #unsupported_feeds_urls <= 0 then
+        UIManager:show(InfoMessage:new{
+            text = _("Downloading news finished."),
+            timeout = 1,
+        })
+    else
+        local unsupported_urls = ""
+        for k,url in pairs(unsupported_feeds_urls) do
+            unsupported_urls = unsupported_urls .. url
+            if k ~= #unsupported_feeds_urls then
+                unsupported_urls = unsupported_urls .. ", "
+            end
+        end
+        UIManager:show(InfoMessage:new{
+            text = T(_("Downloading finished. Could not process some feeds. Unsupported format in: %1"), unsupported_urls)
```

Comment:
I'd say "Downloading news finished." to keep it consistent with the other message.